mods/seasonal/moves.js

```diff
@@ -1043,6 +1067,35 @@ let BattleMovedex = {
 		target: "normal",
 		type: "Fire",
 	},
+	// Osiris
+	nightmarch: {
+		basePower: 60,
+		basePowerCallback: function (pokemon, source, move, target) {
+			let faintedmons = 0;
+			for (const ally of source.side.pokemon) {
+				if (ally.fainted || !ally.hp) faintedmons += 20;
+			}
+			for (const foes of source.side.foe.pokemon) {
+				if (foes.fainted || !foes.hp) faintedmons += 20;
+			}
+			return move.basePower + faintedmons;
+		},
+		accuracy: 100,
+		category: "Physical",
+		desc: "",
+		shortDesc: "",
+		id: "nightmarch",
+		name: "Night March",
+		pp: 5,
+		flags: {protect: 1, mirror: 1},
+		onPrepareHit: function (target, source) {
+			this.attrLastMove('[still]');
+			this.add('-anim', source, "Fire Lash", target);
```

Comment:
Need animation from him, so hold on until i change this

Reply:
☆Osiris: Nightmare + Moongeist Beam + Stomping Tantrum
☆Lycanium Z: Nightmare then moongeist meam then stomping tantrum (I cant do them at the same time)
☆Lycanium Z: ?
☆Lycanium Z: uh
☆Osiris: Yup!
☆Lycanium Z: Ok
☆Lycanium Z: Thanks for letting me know